test-suite/djinni/test.djinni

```diff
@@ -49,3 +49,8 @@ test_helpers = interface +c {
 empty_record = record {
     
 }
+
```

Comment:
Need to merge this with #196 which added the same type, but did so in a separate file.